src/resources/ResourceSnapshots/ResourceSnapshots.ts

```diff
@@ -40,6 +41,19 @@ export default class ResourceSnapshots extends Resource {
         );
     }
 
+    /**
+     * Retrieves a ZIP file holding the content of the target snapshot, in the target format.
+     *
+     * @param {string} snapshotId The unique identifier of the target snapshot.
+     * @param {ExportSnapshotContentOptions} [options]
+     * @returns {Promise<Blob>} A newly created Blob object which contains the zipped snapshot
+     */
+    export(snapshotId: string, options?: ExportSnapshotContentOptions) {
+        return this.api.getFile(this.buildPath(`${ResourceSnapshots.baseUrl}/${snapshotId}/content`, options), {
+            headers: {accept: 'application/zip'},
+        });
+    }
```

Comment:
I'm wondering here, what should we do about `options`?

The API behaviour is to default `contentFormat`.
Should we reflect that defaulting in the code of the `platform-client` with for e.g. a `defaultOption` variable, or leave that entirely to the backend?

How did you proceed so far @coveo/admin-console for the other calls?

Reply:
So far we have let the backend handle default values. I guess that is mainly because the swagger of each service is not always clear about those. One thing that we could do here though is specify the default in the jsdocs.